docs/source/parameters.rst

```diff
@@ -1,6 +1,6 @@
 Snare command line parameters
 =============================
-snare.py [**--page-dir** *folder* ] [**--list-pages**]
+snare [**--page-dir** *folder* ] [**--list-pages**]
```

Comment:
we have 1 param left here - host-ip, could you please add it?

Reply:
I have added it :)